software/deploy/deploy.go

```diff
@@ -28,11 +29,14 @@ var (
 )
 
 var (
-	errNoWorkspaceID             = errors.New("no workspace id provided")
-	errNoDomainSet               = errors.New("no domain set, re-authenticate")
-	errInvalidDeploymentID       = errors.New("please specify a valid deployment ID")
-	errDeploymentNotFound        = errors.New("no airflow deployments found")
-	errInvalidDeploymentSelected = errors.New("invalid deployment selection\n") //nolint
+	errNoWorkspaceID                        = errors.New("no workspace id provided")
+	errNoDomainSet                          = errors.New("no domain set, re-authenticate")
+	errInvalidDeploymentID                  = errors.New("please specify a valid deployment ID")
+	errDeploymentNotFound                   = errors.New("no airflow deployments found")
+	errInvalidDeploymentSelected            = errors.New("invalid deployment selection\n") //nolint
+	ErrDagOnlyDeployDisabledInConfig        = errors.New("to perform this operation, set both deployments.dagOnlyDeployment and deployments.configureDagDeployment to true in the Astronomer Platform")
```

Comment:
if both configs must be true or false to enable/disable why not just have one config that does both?

Reply:
https://astronomer.slack.com/archives/C04JG8563HR/p1704911759959869?thread_ts=1704817127.020259&cid=C04JG8563HR